staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go

```diff
@@ -360,3 +374,31 @@ func (s *WatchServer) HandleWS(ws *websocket.Conn) {
 		}
 	}
 }
+
+func recordWatchListLatency(ctx context.Context, reqScope *RequestScope, metricsScope string) {
```

Comment:
Move it to pkg/endpoints/metrics/metrics.go - it's where a bunch of Record* functions are sitting.